docs/external-id/customers/concept-native-authentication.md

```diff
@@ -49,9 +49,16 @@ The following table compares the two authentication methods to help you decide t
 | **Applicability**  | Suitable for workforce, B2B, and B2C apps, it can be used for native apps, single-page applications, and web apps. | For customer first-party mobile apps, when the same entity operates the authorization server and the app and the user perceives them both as the same entity.|
 | **Go live effort** |  Low. Use it straight out of the box.  |High. The developer builds, owns, and maintains the authentication experience. |
 | **Maintenance effort** | Low. |High. For each feature that Microsoft releases, you need to update the SDK to use it.  |
-| **Security** | Most secure option. |Security responsibility is shared with developers, and best practices need to be followed. It's prone to phishing attacks. |
+| **Security** | Most secure option. |Security responsibility is shared with developers, and [best security practices](#best-security-practices) need to be followed. |
 | **Supported languages and frameworks** | <ul><li>ASP.NET Core</li><li>Android (Kotlin, Java)</li><li>iOS (Swift, Objective-C)</li><li>JavaScript</li><li>React</li><li>Angular</li><li>Nodejs</li><li>Python</li><li>Java</li></ul>  |<ul><li>Android (Kotlin, Java)</li><li>iOS (Swift, Objective-C)</li></ul> For other languages and platforms, you can use our [native authentication API](../../identity-platform/reference-native-authentication-overview.md?bc=/entra/external-id/customers/breadcrumb/toc.json&toc=/entra/external-id/customers/toc.json).  |
 
+## Best security practices
+When using the native authentication APIs:
+- **Store tokens securely**: Tokens should be treated as other credentials and need to be stored safely. Specifically, refresh token, which is a long-lived credential that can be used to obtain new access tokens when it expires. 
```

Comment:
To be fair, it's important to note that these practices should be applied not only when utilizing the native auth API but also with the browser delegated API. Should we still mention those in this case? Maybe, I'm overlooking something.

Reply:
If it applies to both native auth and the browser delegated auth, in my opinion, I don't think we need it here.

Reply:
Thanks, let's see what Kaushik says about it

Reply:
I don't think there are browser delegated APIs available right now. Do you mean MSAL SDK? 
IMO the list here is trying to bring up these best practices in context of native authentication. I think this should be in native auth documentation. Now I am certain we should not put this in the concept level page. Most of these risks/practices are API related and should be in native auth API landing page.

Idea is that for anyone wanting to use the API to build the apps should follow these best practices. We should end that message with something like- use our MSAL SDK for some of these best practices already incorporated.

Reply:
> I don't think there are browser delegated APIs available right now. Do you mean MSAL SDK?
My bad. This practice only applies to native auth APIs.

@kaushikkislay do you suggest moving these best security practices in the API section and just link them from the table before this section?